build/hellobuild.md

```diff
@@ -0,0 +1,140 @@
+
+* `RUN pip install flask`
+
+    This second `RUN` instruction requires that we’ve installed pip in the layer before. After applying the previous directive, we can use the pip command to install the flask web framework. This is the framework we’ve used to write our basic “Hello World” application from above, so to run it in Docker, we’ll need to make sure it’s installed. <br/>See also the [RUN instruction](../engine/reference/builder/#run) page in the Dockerfile reference.
+
+* `COPY hello.py /`
+
+    This COPY instruction copies our `hello.py` file from the build’s context local directory into the root directory of our image. After this executes, we’ll end up with a file called `/hello.py` inside the image, with all the content of our local copy! <br/> See also the [COPY instruction](../engine/reference/builder/#copy) page in the Dockerfile reference.
+    
+
+* `ENV FLASK_APP=hello` 
+
+    This ENV instruction sets a linux environment variable we’ll need later. This is a flask-specific variable, that configures the command later used to run our `hello.py` application. Without this, flask wouldn’t know where to find our application to be able to run it. <br/>See also the [ENV instruction](../engine/reference/builder/#env) page in the Dockerfile reference.
+
+
+* `EXPOSE 8000` 
+
+    This EXPOSE instruction marks that our final image has a service listening on port 8000. This isn’t required, but it is a good practice, as users and tools can use this to understand what your image does. <br/>[EXPOSE instruction](../engine/reference/builder/#expose) page in the Dockerfile reference.
+
+* `CMD flask run --host 0.0.0.0 --port 8000`
+
+    This CMD instruction sets the command that is run when the user starts a container based on this image. In this case we’ll start the flask development server listening on all hosts on port 8000. <br/>[CMD instruction](../engine/reference/builder/#cmd) page in the Dockerfile reference.
+
+## Test the example
+
+Go ahead and try this example in your local Docker installation or you can use Docker Play that provides you with a temporary Docker instance on the cloud.
+
+To test this example:
+1. Create a file hello.py with the content of sample A.
+2. Create a file named Dockerfile without an extension with the contents of sample B.
+3. From your Docker instance build it with `docker build -t test:latest .`
+
+    Breaking down the docker build command:  
+    * **`-t`** option specifies the name (required) and tag (optional) of the image we’re building.  
```

Comment:
```suggestion
    * **`-t test:latest`** option specifies the name (required) and tag (optional) of the image we’re building.  
```
Think it reads easier if we group the flag with it's argument to clarify how they're connected together.